app/models/reason.rb

```diff
@@ -0,0 +1,3 @@
+class Reason < ActiveRecord::Base
```

Comment:
@colinbruce is off today. And we already discussed it in person yesterday.

We're not future proofing this as it's difficult to predict the future. What I believe that we're doing here, is that we're creating a model (in the modelling sense, not ActiveRecord's use of model) of something that can be easily extended if need be.

As for:

> For example now can create multiple reasons for 1 application, which is a case we'll have to handle as well.

I haven't see this anywhere, happy to be proved wrong.
With the current setup of the Evidence form object the user should only ever have one reason. If they choose to add another reason, they can just append it to the existing **explanation** attribute.

> Also the user will have to edit the existing reason, which means the form has to pull data from 2 tables.

And that is bad because?